assets/app/scripts/controllers/create/createFromImage.js

```diff
@@ -137,7 +142,14 @@ angular.module("openshiftConsole")
                         message: "Cannot create " + objectName + ". ",
                         details: failure.data.message
                       });
-                      hasErrors = true;
+                    }
+                  );
+                  result.success.forEach(
+                    function(success) {
+                      alerts.push({
+                        type: "success",
+                        message: "Created " + success.kind.toLowerCase() + " \"" + success.metadata.name + "\" successfully. "
```

Comment:
Fixed to humanize `kind` and `objectName`. Notice that eventually resource names can come as part of an api failure message, and we display it as it comes through `failure.data.message`. But for everything else we are covered.